std/functional.d

```diff
@@ -1814,14 +1813,33 @@ if (isCallable!(F))
     {
         return fp;
     }
+    else static if (is(F Func == Func*) && is(Func == function) && is(Func Params == __parameters))
+    {
+        // https://issues.dlang.org/show_bug.cgi?id=24007 - cannot specify linkage on function literal:
```

Comment:
this should be updated to the corresponding GH issue number if possible

Reply:
Oh, indeed. Luckily, I’ve updated this already locally. Let me just remove my debugging code…